manifest_staging/deploy/gatekeeper.yaml

```diff
@@ -627,6 +627,27 @@ spec:
   - secret
   - downwardAPI
 ---
+apiVersion: policy/v1beta1
```

Comment:
The upgrade pod is purely for Helm right? Is there an easy way not to include the extra PSP for the flat manifest?

Reply:
Good catch. I moved it to the chart only.